pkg/controller/build/helpers.go

```diff
@@ -331,11 +332,11 @@ func GetImageBuilderType(cm *corev1.ConfigMap) (string, error) {
 		return defaultBuilder, nil
 	}
 
-	validImageBuilderTypes := sets.NewString(OpenshiftImageBuilder, CustomPodImageBuilder)
+	validImageBuilderTypes := sets.NewString(string(OpenshiftImageBuilder), string(CustomPodImageBuilder))
```

Comment:
suggestion: You can use generics to construct an `ImageBuilderType` set here instead of having to cast to / from `string` to look things up:

```golang
validImageBuilderTypes := sets.New[ImageBuilderType](OpenshiftImageBuilder, CustomPodImageBuilder)

if !validImageBuilderTypes.Has(configMapImageBuilder) {
    // ...
}
```